case-studies/aepsi.html

```diff
@@ -0,0 +1,123 @@
+            <div class="hero__right-img">
+              <img src="/images/logo/logos-featured-clients/aepsi.png" alt="AEPSi Logo">
+            </div>
+          </div>
+        </div>
+    </section>
+    <div class="content-page-wrapper">
+      <section class="section content-block">
+        <div class="container">
+         
+        <ul>
+            <li>Developed a robust early childhood assessment web application using Phoenix LiveView for real-time interactivity and streamlined data management.</li>
+            <li>Integrated Oban for background job processing, enabling efficient and uninterrupted data reporting and CSV generation.</li>
+            <li>Employed Tailwind UI to create a responsive and accessible interface, ensuring compliance with <a target="_blank"href="https://www.w3.org/WAI/WCAG2AAA-Conformance">WCAG Standards</a> (AA & AAA).</li>
+            <li>Implemented PostgreSQL for scalable and reliable data storage, comprehensive reporting, and customized assessments that meet diverse educational needs.</li>
+          </ul>
+         
+          <figure>
+            <img src="{{ site.baseurl }}/images/case-studies/aepsi/AEPS3Dashboard.gif" alt="AEPSi Web Application Dashboard"
+              width="100%">
+            <figcaption>AEPSi Web Application Dashboard</figcaption>
+          </figure>
+      
+        </div>
+    </section>
+
+    <div class="content-page-wrapper">
+        <section class="section content-block">
+          <div class="container">
+      <h2>Data-Driven App Development Goals</h2>
+
+      <p>Educators have a crucial role in childhood development, but often face challenges in accurately assessing and supporting each child’s unique progress. AEPS-3 is a comprehensive assessment tool that bridges the gap by providing precise, actionable data.</p>
+
+      <p>The AEPSi web application captures and displays clear data on each child’s growth, enabling educators to tailor their strategies effectively and engage parents in their child's development journey. Brookes Publishing performed <a target="_blank"href="https://aepsinteractive.com/wp-content/uploads/2023/02/AEPS-3-Research_1.23.pdf">extensive research</a> to increase the utility, reliability, and validity of their assessment tools and worked with SmartLogic to apply those findings to the new app design.</p>
+
+      <blockquote>
+        "Without a doubt, I would recommend SmartLogic to anyone looking for a partner to develop applications. I’ve actually worked with SmartLogic for 12 years now. Throughout the years working with them, they bring so much knowledge and expertise."
+        <footer>– Monica Belle, Implementation Manager, Brookes Publishing</footer>
+      </blockquote>
+
+    </div>
+</section>
+
+      <div class="content-page-wrapper">
+        <section class="section content-block">
+          <div class="container">
+
+      <h2>Building a Reliable, User-Friendly Interface with Phoenix LiveView, PostgreSQL, and Tailwind UI</h2>
```

Comment:
We technically use Tailwind CSS to build custom Phoenix components.

```suggestion
      <h2>Building a Reliable, User-Friendly Interface with Phoenix LiveView, PostgreSQL, and Tailwind CSS</h2>
```